style/index_tablet.css

```
@@ -0,0 +1,83 @@
+/* tablet */
+@media (max-width:1199px) {
+    .section-text> br{
+        display:none;
+    }
+    .banner-text > br {
+        display:none;
+    }
```

P3:
아래처럼 작성하셔도 동일 동작합니다.
둘이 비슷한 맥락이라 묶어서 작성하는게 더 좋을 것 같습니다.

```suggestion
    .section-text> br,
    .banner-text > br {
        display:none;
    }
```